cpp/unreal_plugins/Vehicle/Source/Vehicle/Vehicle.Build.cs

```diff
@@ -4,13 +4,13 @@
 
 using UnrealBuildTool;
 
-public class OpenBot : CommonModuleRules
+public class Vehicle : CommonModuleRules
 {
-    public OpenBot(ReadOnlyTargetRules target) : base(target)
+    public Vehicle(ReadOnlyTargetRules target) : base(target)
     {
         SP_LOG_CURRENT_FUNCTION();
 
-        PublicDependencyModuleNames.AddRange(new string[] {"CoreUtils"});
+        PublicDependencyModuleNames.AddRange(new string[] {"ChaosVehicles", "ChaosVehiclesCore", "CoreUtils"});
```

Comment:
Let's move these dependencies into `CommonModuleRules`. It's not strictly necessary, but just for convenience. I like having all the Unreal dependencies in one place.

Reply:
If we do this, we will also need to include `ChaosVehiclesPlugin` in the `.uplugin` files for all our plugins. Otherwise, UE throws 
```bash
2>EXEC : warning : Plugin 'CommonModuleRules' does not list plugin 'ChaosVehiclesPlugin' as a dependency, but module 'CommonModuleRules' depends on 'ChaosVehicles'.
2>EXEC : warning : Plugin 'CoreUtils' does not list plugin 'ChaosVehiclesPlugin' as a dependency, but module 'CoreUtils' depends on 'ChaosVehicles'.
```

Reply:
Ew I don't like that. In that case, you made the right call. I'm not sure why we are allowed to put all of our other Unreal module dependencies in one place except for the `Chaos` ones. Anyway you're right. We lose all potential benefits if we need to update all of our `uplugin` files.

Reply:
I guess if a module is part of an UE plugin and not UE source code, UE expects the plugin name to be added to the dependency list. This is why we are seeing this behavior with `ChaosVehiclesPlugin` and it's modules.